test/irmin-pack/test_gc.ml

```diff
@@ -356,7 +356,7 @@ module Gc = struct
     let* () = S.Repo.close t.repo in
     Alcotest.(check bool)
       "RW cleaned up" false
-      (Sys.file_exists (Filename.concat store_name "store.0.suffix"));
```

Comment:
what do you think about adding a test with gc and split that checks that the correct chunks are removed?

Reply:
Good idea. I was already modifying the migrated v3 store test so I just added them to that one.